src/olympia/stats/utils.py

```diff
@@ -21,10 +23,15 @@ def rows_to_series(rows, filter_by=None):
     """Transforms BigQuery rows into series items suitable for the rest of the
     AMO stats logic."""
     for row in rows:
+        submission_date = (
+            datetime.strptime(row['submission_date'], '%Y-%m-%d').date()
+            if not isinstance(row['submission_date'], date)
+            else row['submission_date']
```

Comment:
I thought you said BigQuery returned `date` instances already?  When are they not `date`s?  (Should there be try... except handling if the data format is incorrect?)

Reply:
right, although this is luck because I didn't specifically check the code of the third-party lib.. There is no date-related error so far so I can remove this for now, I guess.